test/controllers/tasks_controller_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class TasksControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Tests are failing for me - test_should_get_edit, test_should_get_delete, test_should_get_index.

Reply:
Sign up is added, and everything except tests is fixed